test/1_stdlib/FloatingPoint.swift.gyb

```diff
@@ -180,7 +182,9 @@ FloatingPoint.test("Double/staticProperties") {
   expectBitwiseEqual(0x1.921f_b544_42d1_8__p1, Ty.pi)
   expectBitwiseEqual(0x1.0p-52, Ty.ulpOfOne)
   expectBitwiseEqual(0x1.0p-1022, Ty.leastNormalMagnitude)
+#if !arch(arm)
   expectBitwiseEqual(0x1.0p-1074, Ty.leastNonzeroMagnitude)
+#endif
```

Comment:
Sure, it's a copy-paste of the line above. Anyway to pause the "test and merge", or will it just pick up my most recent commit when it merges?